src/core_plugins/kibana/common/tutorials/heartbeat_instructions.js

```diff
@@ -146,7 +142,7 @@ export const createHeartbeatInstructions = context => ({
         defaultMessage: 'The `setup` command loads the Kibana dashboards. If the dashboards are already set up, omit this command.',
       }),
       commands: [
-        'PS C:\\Program Files\\Heartbeat> heartbeat.exe setup',
+        'PS C:\\Program Files\\Heartbeat> .\\heartbeat.exe setup',
```

Comment:
As I copied this from other add data UI code, I assume it's also wrong in all other parts.